src/programs/match_template/match_template.cpp

```diff
@@ -214,9 +215,10 @@ void MatchTemplateApp::DoInteractiveUserInput()
 	padding = my_input->GetFloatFromUser("Padding factor", "Factor determining how much the input volume is padded to improve projections", "1.0", 1.0, 2.0);
 //	ctf_refinement = my_input->GetYesNoFromUser("Refine defocus", "Should the particle defocus be refined?", "No");
 	particle_radius_angstroms = my_input->GetFloatFromUser("Mask radius for global search (A) (0.0 = max)", "Radius of a circular mask to be applied to the input images during global search", "0.0", 0.0);
-//	my_symmetry = my_input->GetSymmetryFromUser("Template symmetry", "The symmetry of the template reconstruction", "C1");
+	my_symmetry = my_input->GetSymmetryFromUser("Template symmetry", "The symmetry of the template reconstruction", "C1");
 #ifdef ENABLEGPU
 	use_gpu_input = my_input->GetYesNoFromUser("Use GPU", "Offload expensive calcs to GPU","No");
+	max_threads = my_input->GetIntFromUser("Max. threads to use for calculation", "when threading, what is the max threads to run", "1", 1);
```

Comment:
This should only be asked if they say yes to use gpu right?

Reply:
So, maybe yes, maybe no? Right now (with the last change made) if a user says "No" to gpu, and ">1" to max_threads, they'll be warned and max_threads will be set to one.

I think this should be preferred to having a (more) variable number of inputs, since this makes scripting easier. I could add a warning at the input line?

Reply:
Ok, whatever you think is best